main.go

```diff
@@ -42,6 +44,11 @@ import (
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
+	Version  = "dev"
```

Comment:
this would be the release version number , right? eg.. `v0.1.3`? do we have to manually change this value for every release?

Reply:
Nope we set it using ldflag when we build